### PR TITLE
created first branch

### PR DESCRIPTION
**Pull request by kingqabil**
## For this project I have done the following:

- [ ] Create a repository,  
- [ ] Set up linters,
- [ ] Add HTML file, 
- [ ] Add CSS file,   
- [ ] Add a test md file,
- [ ] Add descriptive README file.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @kingqabil, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] Add a PR description explaining all the features you have implemented.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflow/linters.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Insights
+linters-config/html-css/.github/workflows/linters.yml
+@SpaYco
+SpaYco Bump up outdated packages (#142)
+Latest commit dfd8253 on Apr 16
+ History
+ 3 contributors
+@MauricioRobayo@SpaYco@bolah2009
+48 lines (45 sloc)  1.45 KB
+
```
Comment (mke2111) [inline .github/workflow/linters.yml:33]: - [ ] Remove this unnecessary piece of code from the linter.yml file. (only the highlighted part. For this reason, your linters are not detected.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by
```
Comment (mke2111) [inline test.md:1]: - [ ] As the file clearly indicates, this file should be added to the ignore file, and it shouldn't exist in your repo file.

File: .github/workflow/linters.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+linters-config/html-css/.github/workflows/linters.yml
+@SpaYco
+SpaYco Bump up outdated packages (#142)
+Latest commit dfd8253 on Apr 16
+ History
+ 3 contributors
+@MauricioRobayo@SpaYco@bolah2009
+48 lines (45 sloc)  1.45 KB
+  
+name: Linters
```
Comment (mke2111) [inline .github/workflow/linters.yml:34]: - [ ] Use the original linter file provided by microverse and you should not tamper with its contents. Ensure that the linters are detected and they pass with a green tick.

File: .github/workflow/linters.yml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Privacy
+Security
+Status
+Docs
+Contact GitHub
+Pricing
+API
+Training
+Blog
+About
```
Comment (mke2111) [inline .github/workflow/linters.yml:93]: - [ ] These lines shouls also not be apart of this file. Use the original file.

File: README.md [hunk 5]
```diff
@@ -1,59 +1,36 @@
...
-- Major languages
-- Frameworks
-- Technologies used
+- HTML and CSS
+- GitHub Flow
+- Linters
 
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
Comment (mke2111) [inline README.md:22]: - [ ] Add the correct live demo link for your page,

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @kingqabil, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] You have not implemented all the changes that were previously requested. Let me know if you need more clarification.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by icebox827 — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**